qa/workunits/cephtool/test.sh

```diff
@@ -1682,7 +1682,7 @@ function test_mon_osd_misc()
   ceph pg set_full_ratio 95 2>$TMPFILE; check_response 'not in range' $? 22
 
   # expect "not in range" for invalid overload percentage
-  ceph osd reweight-by-utilization 80 2>$TMPFILE; check_response 'higher than 100' $? 22
+  ceph osd reweight-by-utilization 80 2>$TMPFILE; check_response 'not in range' $? 22
```

Comment:
i have following output locally
```
$ ./bin/ceph osd reweight-by-utilization 80
*** DEVELOPER MODE: setting PATH, PYTHONPATH and LD_LIBRARY_PATH ***
2017-03-03 16:14:34.543843 7f76bc7ed700 -1 WARNING: all dangerous and experimental features are enabled.
2017-03-03 16:14:34.553702 7f76bc7ed700 -1 WARNING: all dangerous and experimental features are enabled.
Error EINVAL: You must give a percentage higher than 100. The reweighting threshold will be calculated as <average-utilization> times <input-percentage>. For example, an argument of 200 would reweight OSDs which are twice as utilized as the average OSD.
```

which is expected. see `OSDMonitor::reweight_by_utilization()`. i am not sure why we should expect `not in range` instead.